setup/web.sh

```diff
@@ -122,6 +122,17 @@ cat conf/mozilla-autoconfig.xml \
 	 > /var/lib/mailinabox/mozilla-autoconfig.xml
 chmod a+r /var/lib/mailinabox/mozilla-autoconfig.xml
 
+# Create a generic mta-sts.txt file which is exposed via the
+# nginx configuration at /.well-known/mta-sts.txt
+# more documentation is available on: 
+# https://www.uriports.com/blog/mta-sts-explained/
+
```

Comment:
```suggest
PUNY_PRIMARY_HOSTNAME=$(echo "$PRIMARY_HOSTNAME" | idn2)
```

we'd need to `apt install idn2` before here...

Does MiaB punycode the host name currently for A and MX records say I called it `ボックス.ポスト.com` (box.post.com) would MiaB hande other records as `xn--pckua2a7g.xn--zck4a9c.com`  ?

Reply:
Everything is punycoded, yes. I have a test domain https://ツ.life/ running on my Mail-in-a-Box to make sure that works.

Reply:
Rather than installing idn2, we should already be getting the idna Python package via https://github.com/mail-in-a-box/mailinabox/blob/master/setup/questions.sh#L18 so we could also write a small Python script to do this.

Reply:
I used the idn2 package; the reason is, we doesn't use idn2 anywhere else in the application and this script is a shell script.

Thanks, both, for the feedback.